scripts/setup-release-on-ubuntu.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+RELEASE_VERSION=${RELEASE_VERSION:=0.24.0}
+RELEASE_FILE="https://github.com/aeternity/epoch/releases/download/v${RELEASE_VERSION}/epoch-${RELEASE_VERSION}-ubuntu-x86_64.tar.gz"
+TEMP_RELEASE_FILE=${TEMP_RELEASE_FILE:=/tmp/epoch.tgz}
+TARGET_DIR=${TARGET_DIR:=$HOME/epoch}
+EPOCH_CONFIG=${EPOCH_CONFIG:=$TARGET_DIR/epoch.yaml}
+NPROC=$(nproc)
+
+echo ""
+read -sp "Please provide a secret key passphrase to be used for your keys: " keypassphrase
+echo ""
+
+echo -e "\nPrepare host system and install dependencies...\n"
+sudo apt-get update
+sudo apt-get upgrade -y
+sudo apt-get install curl jq tmux htop libsodium23
```

Comment:
why do we need to install curl, jq, tmux and htop? Seems to me that this is just user preference and not needed in this guide.

 I assume an `-y` would be beneficial here.

Reply:
Sure, `curl` is required for getting the archive. The rest is optional.